Gemfile.rails_version

```diff
@@ -2,13 +2,16 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV["RAILS_VERSION"] || '4.2.10'
+env_rails_version = '5.1.7'
```

Comment:
i set this manually because otherwise it wouldn't use the target 5.1 version

Reply:
Which version would it use?

Reply:
it kept using the 4.2.10, iirc

Reply:
I wonder if that's due to the gemspec file below.

Reply:
Errr --- you may have to set this ENV var in whatever directory you're running this in. Assuming you're running the tests here. Could this just be changed to 
```suggestion
env_rails_version = ENV['RAILS_VERSION'] || '5.1.7'
```